docs/core-concepts/components.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Components
+---
+
+This document provides an overview of the components required for a fully functional and operational Karmada setup.
+
+![components](../resources/general/components.png)
+
+## Control Plane Components
+
+A complete and working Karmada control plane consists of the following components. 
+The `karmada-agent` might be optional, that depends on 
+[Cluster Registration Mode](https://kubernetes.io/docs/concepts/overview/components/).
+
+### karmada-apiserver
+
+The API server is a component of the Karmada control plane that exposes the Karmada API in addition to the Kubernetes API. 
+The API server is the front end of the Karmada control plane.
+
+Karmada API server directly uses the implementation of `kube-apiserver` from Kubernetes, which is the reason why Karmada 
```

Comment:
If we directly state here that we are using the LTS version of kube-apiserver, wouldn't it be more explicit?